src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -494,10 +494,24 @@ void AirspeedModule::update_params()
 		param_get(_param_handle_fw_thr_max, &_param_fw_thr_max);
 	}
 
+	const float prev_scale[MAX_NUM_AIRSPEED_SENSORS] = {
+		_param_airspeed_scale[0],
+		_param_airspeed_scale[1],
+		_param_airspeed_scale[2]
+	};
+
 	_param_airspeed_scale[0] = _param_airspeed_scale_1.get();
 	_param_airspeed_scale[1] = _param_airspeed_scale_2.get();
 	_param_airspeed_scale[2] = _param_airspeed_scale_3.get();
 
+	for (int i = 0; i < MAX_NUM_AIRSPEED_SENSORS; i++) {
+		if (fabsf(_param_airspeed_scale[i] - prev_scale[i]) > FLT_EPSILON) {
```

Comment:
This will also trigger when the parameter is [set by the system](https://github.com/PX4/PX4-Autopilot/blob/482683d156f66d4ea3f2bfd8fc340b0fe3efd385/src/modules/airspeed_selector/airspeed_selector_main.cpp#L452) and not by the user, correct?